pyproject.toml

```
@@ -26,6 +26,7 @@ packages = [{ include = "apps" }]
 
 [tool.poetry.scripts]
 pyappsutils-unusedcode = "apps.unused_code.unused_code:get_unused_functions"
+pyappsutils-polarion-tc-requirements = "apps.polarion.polarion_verify_tc_requirements:has_verify"
```

```suggestion
pyappsutils-polarion-verify-tc-requirements = "apps.polarion.polarion_verify_tc_requirements:has_verify"
```

the command needs to indicate that this is about verifying 

also, please shorten `pyappsutils` to something like `pyutils`

---

This was named based on review comments. Now it is being used in cnv-tests. I would prefer to keep it as `pyappsutils`.

---

added `verify`.

---

I will just shorten it as you adviced. I agree it is annoying to have a long name.